.github/workflows/build-test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/package.lock') }}
+          key: ${{ runner.os }}-modules-${{ hashFiles('./package-lock.json') }}
```

Comment:
`hashFiles` is failing now to hash all package-lock's. Probably its because we have too much of them. I think we can cashe only root package-lock, because it is enaugh to ensure same installation